default.nix

```diff
@@ -38,10 +38,10 @@ in {
              (compiler.callPackage ./iohk/default.nix {})
              (drv: {
                 executableToolDepends = [ pkgs.makeWrapper ];
-                libraryHaskellDepends = iohk-ops-extra-runtime-deps;
+                libraryHaskellDepends = iohk-ops-extra-runtime-deps ++ [ cardano-sl-pkgs.cardano-sl-auxx pkgs.file nixops ];
```

Comment:
Why are these needed to be in nix-shell?

Reply:
auxx and file is for the update proposals
keygen is for dev cluster deployments